serverless.yml

```diff
@@ -146,7 +146,15 @@ functions:
           enabled: ${env:ENABLE_SYNC_LEGACY, self:custom.false}
     environment:
       GRAPHQL_ENDPOINT: ${self:custom.GRAPHQL_ENDPOINT}
-
+  pushNotifications:
+    handler: handlers/pushNotificationHandler.handle
+    timeout: 60
+    events:
+      - schedule:
+          rate: cron(0 12 * * ? *)
```

Comment:
serverless allows us to use rate() syntax instead, this line can probably just be:
`rate: rate(12 hours)`

Reply:
Watershed wanted emails sent every morning at 6AM. cron(0 12 * * ? *) runs the handler at 12:00 UTC, which is 6AM Texas time (um, sometimes...).